send_to_telemetry.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python3
+
+from dataclasses import asdict, dataclass
+import json
+import sys
+
+
+@dataclass
+class TestResult:
+    appName: str
+    channel: str
+    version: str
+    url: str
+    metrics: dict
+
+
+def main(path):
+    with open(path) as f:
+        data = json.load(f)
+
+    with open("wpt.json") as f:
+        metrics = json.load(f)
+
+    for test in data:
+        sample = test["data"]["median"]["firstView"]
+        metrics = {
+            "TTFB": sample.TTFB,
+            "render": sample.render,
+            "firstPaint": sample.firstPaint,
+            "timeToContentfulPaint": sample.timeToContentfulPaint,
+            "domContentFlushed": domContentFlushed,
```

Comment:
@stephendonner  is this line missing `sample.`?

Reply:
OMG yes; can't believe I missed that...

Reply:
Thx; fixing now!